### DISCUSSION TITLE
Modified RSI -- RSI of EMA(14, close)

### DISCUSSION THREAD

**Discussion by rkravuri** [DaveSkender/Stock.Indicators#998] [Q&A] [answered]
Hi Team,

There is a way in TradingView to configure the data Source on the RSI indicator. The Source can be Open, High, Low, Close, etc, or even the output of another indicator plotted on the chart. 

I want to evaluate the RSI of EMA(14, close). 

Please help me understand how I can achieve this in Stock.Indicators ?

**Comment by DaveSkender** [comment, marked as answer]
``` csharp
var results = quotes
  .Use(CandlePart.Close)
  .GetRsi(14);
```
The default uses close price (e.g. `quotes.GetRsi(14)`), so you'd really only need to do this alternate selection if you wanted something else.  You can also use composite candle parts, like HL2.  More information: [utilities section of our guide](https://dotnet.stockindicators.dev/utilities/#use-alternate-price).

If you're trying to chain something, like an RSI of an EMA using HL2 prices:

``` csharp
var results = quotes
  .Use(CandlePart.HL2)
  .GetEma(14)
  .GetRsi(14);

// or with default close price
var results = quotes
  .GetEma(14)
  .GetRsi(14);

// or with multiple steps if you want EMA separately
var emaResults = quotes
  .GetEma(14);

var rsiEmaResults = emaResults
  .GetRsi(14);
```

**Comment by rkravuri** [comment]
Thank you, @DaveSkender. 

This has helped me to progress in automating a strategy based on Modified RSI.
